packages/perseus/src/server-item-renderer.tsx

```diff
@@ -60,10 +60,24 @@ type DefaultProps = Required<
 >;
 
 type State = {
+    /**
+     * questionCompleted is used by Radio and LabelImage to change what
+     * it renders after a learner has checked the correctness of an answer.
+     * For instance LabelImage disables inputs for correct answers and marks
+     * incorrect answers so learners can update what they submitted.
```

Comment:
I like referencing widgets that use the flag as an example, but I think we can word this more generically (like, what _should_ this flag be used for). :) 

So this flag is used by the renderer to signal to widgets that the question is done and they can alter what they render to reveal explanations or other information about why the learner's input was wrong, etc. 

Your comment is basically good, I guess I would just suggest that we remove the wording that implies that only Radio and LabelImage use this (they do! but in the future they may not be the only ones).  Sorry, I'm probably splitting hairs.

Reply:
Also, it might be good to add a comment something like this to the `WidgetProps<T>` type as it has this flag too and it'd be helpful within widgets to be able to see that prop's intended usage.